src/DynamoCoreWpf/Views/Preview/PreviewControl.xaml.cs

```diff
@@ -307,6 +323,22 @@ private void RefreshCondensedDisplay(Action refreshDisplay)
 
             string newContent = "null";
 
+            if (mirrorData is ProtoCore.MirrorDataInProgress
+                // When CBN is created, its cached value is set to MirrorDataInProgress.
+                // And it's not updated after update graph task is complited.
+                && !((nodeViewModel.NodeLogic is CodeBlockNodeModel) && (nodeViewModel.NodeLogic as CodeBlockNodeModel).Code == String.Empty))
```

Comment:
I think the crash happens on all nodes (though other kinds of nodes are not so easy to reproduce), not only to code block node.